elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -299,4 +325,22 @@ public Integer getQueryLimit() {
     public void cancel() {
         session.cancelQuery();
     }
+
+    @Override
+    public QueryDetail explain(EntityProjection projection, RequestScope scope) {
+        Class<?> entityClass = projection.getType();
+        String modelName = entityClass.getName();
+        List<String> queryTextList = getQueryText();
```

Comment:
I don't think this is correct.  The explain interface wants to explain _only_ this entity projection.  The getQueryText function is returning everything to date that has happened with this projection.